main.go

```diff
@@ -132,6 +132,7 @@ func main() {
 	log.Infoln("Build context", version.BuildContext())
 
 	// Bail early if the config is bad.
+	var err error
```

Comment:
It confuse me, because I thought that if I use := operator, then new local variable will be declared and in that case sc.C will not be filled, or compilation error will occur.